composer/packages/vs-syntax-tree/src/components/dropdown/detailsArrayCard.tsx

```diff
@@ -0,0 +1,59 @@
+"use strict";
+/**
+ * Copyright (c) 2021, WSO2 Inc. (http://www.wso2.org) All Rights Reserved.
+ *
+ * WSO2 Inc. licenses this file to you under the Apache License,
+ * Version 2.0 (the "License"); you may not use this file except
+ * in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing,
+ * software distributed under the License is distributed on an
+ * "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY
+ * KIND, either express or implied. See the License for the
+ * specific language governing permissions and limitations
+ * under the License.
+ *
+ */
+import React from "react";
+import { MINUTIAE } from "../../resources/constants";
+import { DetailsArrayCardProp } from "../../resources/tree-interfaces";
+import * as styles from "../../styles/dropdown-tree.styles";
+
+function DropdownArrayDetails(props: DetailsArrayCardProp) {
+    return (
```

Comment:
We can use the [destructuring](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Operators/Destructuring_assignment) as below for better readability and ease of use. i.e, we can avoid repetitively using `props.` and will be easier for renaming variables/fields when required.
```suggestion
    const {title, value, type} = props;
    return (
```
Then, we can directly use `title`, `value` and `type` wherever needed.